code/game/machinery/doors/firedoor.dm

```diff
@@ -36,6 +36,14 @@
 	if(!density)
 		return ..()
 	return 0
+	
+/obj/machinery/door/firedoor/ex_act(severity)
+	switch(severity)
+		if(1.0)

```

Comment:
Strangely enough we use this in most of our ex_act procs. Not sure if I'll change it in that case for consistency (that way we can just mass update them at some point anyway).

Good catch though!